src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/CommentsIndentation/context/case3/Test.java

```diff
@@ -0,0 +1,22 @@
+package TreeWalker.CommentsIndentation;
+
+public class Test {
+    public void test() {
+        try {
+        int a;
+            // violation without filter
+        } catch (Exception ex) {}
```

Comment:
Add another `try` into this method with a suppresed violation to show that filter covers only current SLIST

Reply:
done